### DISCUSSION TITLE
抽象高中生活

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#4968] [灌水]
我们班同学也是唐到没边，我在学校里面热的慌，就把衣服撩起来一点，然后就被女班长拍照，后面就开始叽叽歪歪，也是无语上了。
这就算了，关键一帮人搁哪拿我当傻子，我嘞个大唐盛世啊

**Comment by March-777th** [comment]
直接拍照啊🤮这似慕母狗也是逆天，就当是路边一条野狗，别跟她计较

**Comment by manyshoes** [comment]
你发这不如发去贴吧挂人
